gems/pending/appliance_console.rb

```diff
@@ -111,11 +114,14 @@ module ApplianceConsole
 
   loop do
     begin
-      host     = LinuxAdmin::Hosts.new.hostname
-      ip       = Env["IP"]
-      mac      = Env["MAC"]
-      mask     = Env["MASK"]
-      gw       = Env["GW"]
+      eth0.reload
+      eth0.parse_conf
+
+      host     = LinuxAdmin::Hosts.new.hostname.strip
```

Comment:
I think LinuxAdmin should take care of stripping this for you.  What do you think?